pkg/cmd/cli/describe/describer.go

```diff
@@ -146,6 +146,9 @@ func (d *BuildDescriber) Describe(namespace, name string) (string, error) {
 
 		return nil
 	})
+
+	// REVIEW: we might want to include information from p.TriggeredBy to the
+	// output of `oc describe`.
```

Comment:
by this PR, triggeredby will describe the build trigger, i think we can ignore these comments for now at least. i'll push an update with updated notes and all.